ovs_performance.py

```diff
@@ -845,20 +848,41 @@ def get_active_datapath_flows():
 #
 def warm_up_verify(requested_flows, timeout):
     run_time = 0
+    wait_time = 0
     active_flows = 0
+    error_continue = 0
 
     while active_flows < requested_flows:
         run_time += 1
         if timeout != 0 and run_time >= timeout:
-            lprint("ERROR: Failed to complete warm-up in time ({} seconds)!".
-                   format(timeout))
-            sys.exit(-1)
+	    if config.warm_up_no_fail:
```

Comment:
I do not think this part should be handled in this function, it should not make assumptions about stopping streams, VMs etc. due to the use case in other tests. I think it should be handled in the test case itself.

Reply:
Thank you for review. I'll send another pull request after making changes.